### PR TITLE
Document setting gateway poll interval

### PR DESCRIPTION
**Pull request by StephenBarlow**
Followup from #1336

### CODE REVIEW

File: docs/source/managed-federation/uplink.mdx
Comment (trevor-scheer) [inline docs/source/managed-federation/uplink.mdx:74]: I have a PR open (should be landing soon) which deprecates this in favor of `pollIntervalInMs`. Can this PR wait for that change to land? :)

**Review by trevor-scheer — APPROVED** [review]

File: docs/source/managed-federation/uplink.mdx
Comment (trevor-scheer) [reply docs/source/managed-federation/uplink.mdx:74]: Also you _can_ just target my branch and merge this into my PR if you'd like! It'll conveniently get ported to 2.x as well.

`trevor/improve-supergraphSdl`

Comment (StephenBarlow) [reply docs/source/managed-federation/uplink.mdx:74]: Ah yeah sure! Will edit and retarget.

**Comment by StephenBarlow** [comment]
Closing in favor of #1366

**Pull request closed without merging**
